### PR TITLE
Add slideshow to show time lapses

### PR DESCRIPTION
**Pull request by Prashantneu8848**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -46,7 +46,59 @@ <h2>About Me </h2>
...
+      <li data-target="#timelapse-slideshow" data-slide-to="1"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="3"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="4"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="5"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="6"></li>
+    </ol>
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/1.mov" type="video/mp4" />
```
Comment (sehtia-raghav) [inline portfolio/src/main/webapp/index.html]: Something to keep in mind when including multimedia - it's nice to have a fallback or alternative - like text representing the content. This is especially helpful for accessibility needs. 
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

You don't need to add a fallback for each vid unless you'd like, but more of an FYI  :)

**Review by sehtia-raghav — COMMENTED** [review]
LG. Pending approval so Cory can LG as well

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Added a fallback as in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
 <div id="timelapse">
   <h2>Timelapses</h2>
   <p>
-    Here are some videos that I have taken and edited
+    Here are some videos that I have taken and edited.
+  <div id="timelapse-slideshow" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#timelapse-slideshow" data-slide-to="0" class="active"></li>
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: Consider removing optional `</li>`. See https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
...
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/6.mov" type="video/mp4" />
+        </video>
+      </div>
+    </div>
+    <a class="carousel-control-prev p-3 mb-2 bg-dark text-white" href="#timelapse-slideshow" role="button"
+      data-slide="prev">
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: Indent 4 spaces when attributes wrap to the next line. See https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
...
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="carousel-control-next p-3 mb-2 bg-dark text-white" href="#timelapse-slideshow" role="button"
+      data-slide="next">
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: Indent 4 spaces when attributes wrap to the next line. See https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -75,14 +75,14 @@ body {
...
-
 #about-me,
 #timelapse,
 #pictures,
 #connect {
   padding: 40px;
   text-align: center;
 }
+
+.carousel-inner img, .carousel-inner video {
```
Comment (corycook) [inline portfolio/src/main/webapp/style.css]: Separate selectors with new lines. See https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -75,14 +75,14 @@ body {
   padding: 15px;
 }
```
Comment (corycook) [inline portfolio/src/main/webapp/style.css:77]: Remove leading 0 on line 63 (GitHub doesn't let me comment that far away from your changes). See https://google.github.io/styleguide/htmlcssguide.html#Leading_0s

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
...
+      <li data-target="#timelapse-slideshow" data-slide-to="4"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="5"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="6"></li>
+    </ol>
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/1.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: This should be before `</video>`.

```html
<video class="video-fluid" autoplay loop muted>
  <source src="./resources/videos/1.mov" type="video/mp4" />
  Sorry the embedded videos is not supported by your browswer.
</video>
```

Comment (corycook) [reply portfolio/src/main/webapp/index.html]: Nit: "video"

Comment (corycook) [reply portfolio/src/main/webapp/index.html]: Nit: "browser"

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
...
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/4.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/5.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: This should be in the `<video>`

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
...
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/3.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/4.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: This should be in the `<video>`

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
...
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/2.mov" type="video/mp4" />
+        </video>
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/3.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: This should be in the `<video>`. Nit: "video" "browser"

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (corycook) [reply portfolio/src/main/webapp/index.html]: Nit: remove optional closing mark `/`

```html
<source src="resources/videos/1.mov" type="video/mp4">
```

**Review by corycook — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 10 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 9 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 8 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 7 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/style.css (hunk 6 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/style.css:77]: Made Changes

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/style.css]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Prashantneu8848) [reply portfolio/src/main/webapp/index.html]: Made Changes

**Review by corycook — APPROVED** [review]
LGTM

**Pull request merged**